msal/src/main/java/com/microsoft/identity/client/claims/RequestedClaimAdditionalInformation.java

```diff
@@ -46,6 +46,27 @@
     @SerializedName(SerializedNames.VALUES)
     private List<Object> mValues = new ArrayList<>();
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof RequestedClaimAdditionalInformation)) return false;
+
+        RequestedClaimAdditionalInformation that = (RequestedClaimAdditionalInformation) o;
+
+        if (mEssential != null ? !mEssential.equals(that.mEssential) : that.mEssential != null)
+            return false;
+        if (mValues != null ? !mValues.equals(that.mValues) : that.mValues != null) return false;
+        return mValue != null ? mValue.equals(that.mValue) : that.mValue == null;
+    }
+
+    @Override
+    public int hashCode() {
+        int result = mEssential != null ? mEssential.hashCode() : 0;
+        result = 31 * result + (mValues != null ? mValues.hashCode() : 0);
+        result = 31 * result + (mValue != null ? mValue.hashCode() : 0);
+        return result;
+    }
```

Comment:
Same here w.r.t to `PMD` and `Checkstyle`

Reply:
Done

Reply:
Done